test/controllers/returnFrequency/ChangeReturnFrequencyConfirmationSpec.scala

```diff
@@ -101,6 +101,7 @@ class ChangeReturnFrequencyConfirmationSpec extends BaseSpec
           lazy val result = {
             mockAuthorise(mtdVatAuthorisedResponse)
             mockContactPreferenceSuccess(ContactPreference("DIGITAL"))
+            mockCustomerDetailsSuccess(circumstanceDetailsModelMax)
```

Comment:
Need to add some tests for new logic in controller as in manage-vat PR, i.e. success vs fail vs null emailVerified boolean